collectors/osd.go

```diff
@@ -0,0 +1,518 @@
+package collectors
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+//OsdCollector sample comment
```

Comment:
Make this comment state the purpose of the `OSDCollector`.  See: https://github.com/digitalocean/ceph_exporter/blob/master/collectors/cluster_usage.go#L28-L31